### ISSUE TITLE
Repository transfer

### ISSUE THREAD

**Issue by Martronic-SA** [collective/collective.github.com#928]
Hello,

I would like to add some modules:
collective.js.multizoom - it just adds the multizoom.js into plone ( 4 and 5 )
collective.z3cform.quantitywidget - a widget that use html5 type="number" with its parameters Plone5
collective.z3cform.phonewidget -  a widget that use html5 type="tel". a validation is provided with python phonenumbers, Plone5
collective.behavior.sql - a new sql behavior for dexterity content types that allow to get data from SQL database into Plone (and save data to SQL db from Plone also), Plone5

Thank you

**Comment by jensens** [comment]
do you want to transfer them from a different github repository or do you want to get them created and you populate it?

**Comment by Martronic-SA** [comment]
I will transfer them from my repository. (They are still not there).

Thanks!

**Comment by jensens** [comment]
Sorry, but looking at https://github.com/Martronic-SA?tab=repositories I can not find them. 
Let me rephrase my question: 

Shall I 

1. create empty repositories so you can populate them or 
2. help moving existing github repositories to the collective?

**Comment by Martronic-SA** [comment]
No I haven't added the modules to my repository yet. I still have to fix some details.
I will tell you once it's transfered so you can move them to the collective.
Ok like that?
Thanks!

**Comment by Martronic-SA** [comment]
Hello again!

Could you move the following repositories to the collective?

[collective.js.multizoom](https://github.com/Martronic-SA/collective.js.multizoom)
[collective.z3cform.quantitywidget](https://github.com/Martronic-SA/collective.z3cform.quantitywidget)
[collective.z3cform.phonewidget](https://github.com/Martronic-SA/collective.z3cform.phonewidget)
[collective.behavior.sql](https://github.com/Martronic-SA/collective.behavior.sql)
[Solgema.RichMenu](https://github.com/Martronic-SA/Solgema.RichMenu)
[Solgema.FlowView](https://github.com/Martronic-SA/Solgema.FlowView)

I would like also to update [collective.js.fullcalendar](https://github.com/collective/collective.js.fullcalendar) which is not Plone 5 compatible and breaks Solgema.fullcalendar. Could you add me to the contributors?

Thank you!

**Comment by gforcada** [comment]
@Martronic-SA all repositories have been forked on collective, you can remove them (if you wish) from your own account.

I sent an invitation to join the collective organization as well.

Thanks for sharing and caring!!

**Comment by gforcada** [comment]
@Martronic-SA forgot to say that, please, close this issue whenever you see that everything requested here is already solved.

**Comment by Martronic-SA** [comment]
Perfect, Thank you.
